scraper.py

```diff
@@ -102,7 +102,7 @@ def login(driver, language, email, password):
   store_login_cookies(driver)
   return True;
 
-def get_categories(driver, language, ignored_categories=[]):
+def get_categories(driver, language, only_categories=None, ignored_categories=[]):
```

Comment:
Might be more clear to call this flag something like `specific_categories`, and maybe make it an empty array by default so it follows the same conventions as `ignored_categories` 😉 

```suggestion
def get_categories(driver, language, specific_categories=[], ignored_categories=[]):
```